### DISCUSSION TITLE
Editting Data

### DISCUSSION THREAD

**Discussion by sam-fakhreddine** [mljar/mercury#473] [Q&A]
### Question
Is there a way to edit data within Mercury?

### Context & Workflow Goal
I’m looking to establish a workflow where:

1. A user uploads a spreadsheet to Mercury.
2. Data is pulled in and displayed on the screen to populate the sheet.
3. A project manager reviews the data, makes updates to any missing or incorrect information, and clicks "Complete."
4. The updated data is saved back to the spreadsheet (XLSX) format, which is then available for download.

Is this type of editable, review-and-update workflow supported in Mercury, or are there any workarounds or recommended methods to achieve it?

**Comment by pplonski** [comment]
Hi @sam-fakhreddine,

We don't have widget with editable sheet. I would love to add one. 

The workaround might be created with current widgets:
- add widget to select cell in the grid
- add text widget to edit the cell value
- add Complete button to save file 

What do you think about it?
